server/tests.go

```diff
@@ -16,9 +19,44 @@ func (ge *TestsEndpoint) AttachHandlers(r *mux.Router) {
 	r.HandleFunc("/ping", ge.ping)
 }
 
+func (ge *TestsEndpoint) checkExternal() bool {
+	isHealthy := true
+
+	if remote_signer.EnableRethinkSKS {
+		conn := database.GetConnection()
+
+		if !conn.IsConnected() {
+			isHealthy = false
```

Comment:
You can just return true here instead of mutating a control variable

Reply:
I was thinking about logging anything, so it should log both services, just not because one was broken the other one should not be verified.

Reply:
Makes sense!